### ISSUE TITLE
Package gtk3 for nimble

### ISSUE THREAD

**Issue by idlewan** [ngtk3/nim-gtk3#1]
I appreciate very much the big effort doing the wrapper. It is though a little annoying to have to either link or copy the files all over the place (if several projects use gtk3).

Is there a specific reason that prevents them from being included in nimble ? Would there be any conflict with current gtk2 packages ? Would it be better to have atk, gdk3, glib, gio and all be part of the same nimble package like gtk2 does, or separate packages ?

**Comment by PMunch** [comment]
I have to agree on this. To make it easier for you can can simply clone the repos, and then run "nimble install" in them to install the repositories into nimble. This means you don't have to copy the files around as you said. But it's still inconvenient to have to clone a bunch of repos when "nimble install gtk3" could fetch all the dependencies automatically.
